### DISCUSSION TITLE
Multiple formats

### DISCUSSION THREAD

**Discussion by agnesnutter** [Tyrrrz/DiscordChatExporter#717] [Q&A]
I'd like to export to json, csv, txt, _and_ html, for different purposes.

Is there any way to achieve this with one command, so the tool doesn't have to crawl every channel four separate times?

**Comment by 96-LB** [comment]
There is not right now. It was attempted at one point, but the feature had to be scrapped due to design/implementation complexities, so it remains an open issue. See #265 and #410.
